.drone.yml

```diff
@@ -24,7 +24,43 @@ steps:
     commands:
       - npm run test
 
-  - name: build
+  - name: build staging
+    image: node:12
+    pull: always
+    environment:
+      DIRECTUS_URL:
+        from_secret: directus_url
+      DIRECTUS_TOKEN:
+        from_secret: directus_token
+    commands:
+      - npm run export:staging
+    when:
+      branch:
+        exclude:
+          - master
+
+  - name: deploy staging
+    image: cupcakearmy/drone-deploy
+    pull: always
+    environment:
+      PLUGIN_KEY:
+        from_secret: key
+    settings:
+      host: love-foundation.org
+      user: root
+      port: 1312
+      target: /srv/staging
+      sources:
+        - ./__sapper__/export
+        - ./docker-compose.prod.yml
```

Comment:
This will overwride the server config.
better to have a separate file for docker-compose

```
#docker-compose.staging.yml

version: '3.7'
networks:
    proxy:
      external: true
services:
    app:
        image: cupcakearmy/static
        volumes:
            - ./__sapper__/export:/srv
        networks:
            - proxy
        labels:
            - "traefik.enable=true"
            - "traefik.port=80"
            - "traefik.docker.network=proxy"
            - "traefik.backend=webnew-staging"
            - "traefik.frontend.rule=staging.love-foundation.org"
```